### PR TITLE
Write perldelta for 8e9664dc9f (:reader attribute on fields)

### PR DESCRIPTION
**Pull request by leonerd**

### CODE REVIEW

File: pod/perldelta.pod [hunk 1]
```diff
@@ -27,6 +27,21 @@ here, but most should go in the L</Performance Enhancements> section.
 
 [ List each enhancement as a =head2 entry ]
 
+=head2 C<:reader> attribute for field variables
+
+When using the C<class> feature, field variables can now take a C<:reader>
+attribute.  This requests that an accessor method is automatically created
```
Comment (khwilliamson) [inline pod/perldelta.pod]: be automatically ...

**Review by khwilliamson — APPROVED** [review]
otherwise lgtm

**Comment by mauke** [comment]
Shouldn't it mention the `:reader(custom_name)` feature, too?

**Comment by leonerd** [comment]
> Shouldn't it mention the `:reader(custom_name)` feature, too?

Imean I guess it could, but the point of perldelta isn't to fully document everything, just to announce that it exists, briefly describe it, and say where to find more. I didn't want to make the entry too long.

**Review by rwp0 — APPROVED** [review]

**Pull request merged**